java/test/jmri/implementation/PackageTest.java

```diff
@@ -47,15 +45,18 @@ public static Test suite() {
         suite.addTest(OffsetHighCvProgrammerFacadeTest.suite());
         suite.addTest(ResettingOffsetHighCvProgrammerFacadeTest.suite());
         suite.addTest(RouteTest.suite());
-        suite.addTest(SE8cSignalHeadTest.suite());
-        suite.addTest(SignalHeadSignalMastTest.suite());
         suite.addTest(SignalSpeedMapTest.suite());
         suite.addTest(SignalSystemFileCheckTest.suite());
         suite.addTest(SingleTurnoutSignalHeadTest.suite());
         suite.addTest(TwoIndexTcsProgrammerFacadeTest.suite());
         suite.addTest(BundleTest.suite());
+        // Converted to JUnit4:
         suite.addTest(new junit.framework.JUnit4TestAdapter(DccConsistTest.class));
+        suite.addTest(new junit.framework.JUnit4TestAdapter(DccSignalHeadTest.class));
```

Comment:
Yes, the suite.addTest() lines should be replaced.

Order needs to be maintained across the entire suite of tests regardless of JUnit syntax being used. Blocking out the JUnit 4 tests separately breaks that order.